main.py

```diff
@@ -93,22 +100,27 @@ def main_fun(num):
         path = input()
         open_file(path)
     else:
-        print("Something is wrong")
+        print("This statement does not exist")
+        print("Try again!")
+        print()
+        time.sleep(1)
 
 
 if __name__ == '__main__':
-    view_menu()
 
     while True:
-        print("Enter number: ")
+        view_menu()
+        print("Enter choice: ")
         choice = input()
         if choice.isdigit():
             choice = int(choice)
         else:
             if choice == 'q' or choice == "Q":
                 sys.exit(0)
             else:
-                print("Something is wrong")
+                print("Unrecognized command")
+                time.sleep(1)
```

Comment:
Maybe it would be better to wait for a user to press a key instead of waiting 1s. Just print: "Unrecognized command! Press enter and try again!".